docs/formats/zson.md

```diff
@@ -651,16 +648,12 @@ where there are at least two types in the list.
 
 #### 3.5.5 Enum Type
 
-An _enum type_ has two forms.  The simple form is:
+An _enum type_ has the form:
 ```
 < <identifier>, <identifier>, ... >
 ```
-and the complex form is:
-```
-< <identifier>:<value>, <identifier>=<value>, ... >
-```
-In the same form, the underlying enum value is equal to its positional
-index in the list of identifiers as an uint64 type.
+The positional index of the each enum symbol is significant, e.g., two enum types
+with the same set of symbols but in different order are distinct.
```

Comment:
Do you feel there's a benefit here over defining enum types with the same set of symbols to be identical? That feels tidier.

Reply:
On the fence... let's merge and we can change it if we want...

Reply:
issue #2823